custom/icds_reports/ucr/tests/test_daily_feeding_daily_attendance.py

```diff
@@ -0,0 +1,85 @@
+from mock import patch
+
+from custom.icds_reports.ucr.tests.test_base_form_ucr import BaseFormsTest
+from decimal import Decimal
+from django.test import override_settings
+
+
+@patch('custom.icds_reports.ucr.expressions._get_user_location_id',
+       lambda user_id: 'qwe56poiuytr4xcvbnmkjfghwerffdaa')
+@patch('corehq.apps.locations.ucr_expressions._get_location_type_name',
+       lambda loc_id, context: 'awc')
+@patch('custom.icds_reports.ucr.expressions.GetAppVersion.get_version_from_app_object',
+       lambda self, item, app_version: 20000)
+class TestDailyFeedingForms(BaseFormsTest):
+    ucr_name = "static-icds-cas-static-daily_feeding_forms"
+
+    @override_settings(SERVER_ENVIRONMENT='icds-cas')
```

Comment:
Shouldn't this be `icds`?

Reply:
`icds` is the real, although if the test is just testing "not india" this should be fine